distribution/docker/build.gradle

```diff
@@ -91,20 +98,79 @@ void addCopyDockerfileTask(final boolean oss) {
   }
 }
 
+Closure createAndSetWritable = {location ->
```

Comment:
Just some code style items: 

1. Let's put a space after all `{` so that `{location` is `{ location`
2. Remove the unnecessary semi-colon on line 102